### PR TITLE
User story 1.3

### PR DESCRIPTION
**Pull request by Stepan0806**
Added user story 1.3, its description and given-when-then acceptance criteria to user_stories.md

### CODE REVIEW

**Comment by Marko-Shevchuk** [comment]
Minor spelling mistakes/inconsistencies: 
* in line "On the "My Account" subpage user can view their contact information...": "user" is to be replaced with "users"
* in all lines containing "side menu": "side menu" might be better generalized as "account navigation menu"
* in line "When: The user clicks on the "Add Payment Method" button and provides the necessary payment information.": possible types of payment methods are not spicified, is another user story for electronic payment required?

File: magento/user_stories.md [hunk 1]
```diff
@@ -2,4 +2,103 @@
...
 
-### 1.1 
\ No newline at end of file
+### 1.1 
+
+### 1.3 As a user, I want to be able to view my account profile so that I can access my contact information, my default billing address, my default shipping address, my orders, my downloadable products, my wish list, my address book, my stored payment methods, and my reviews.
+
+**Description**:
+- Every page on Magento should have a link to the profile page when the user is logged in.
+- The profile page should contain links to several subpages in the side menu: "My Account",
```
Comment (Marko-Shevchuk) [inline magento/user_stories.md:11]: "side menu" -> "account navigation menu" ?

File: magento/user_stories.md [hunk 2]
```diff
@@ -2,4 +2,103 @@
...
\ No newline at end of file
+### 1.1 
+
+### 1.3 As a user, I want to be able to view my account profile so that I can access my contact information, my default billing address, my default shipping address, my orders, my downloadable products, my wish list, my address book, my stored payment methods, and my reviews.
+
+**Description**:
+- Every page on Magento should have a link to the profile page when the user is logged in.
+- The profile page should contain links to several subpages in the side menu: "My Account", 
+"My Orders", "My Downloadable Products", "My Wish List", "Address Book", "Account Information" "Stored Payment Methods", "My Product Reviews".
+- On the "My Account" subpage user can view their contact information and change their password, default billing address, or default shipping address.
```
Comment (Marko-Shevchuk) [inline magento/user_stories.md:13]: Count change "user" to "users"

**Review by Marko-Shevchuk — APPROVED** [review]
Insignificant spelling mistake

File: magento/user_stories.md [hunk 3]
```diff
@@ -2,4 +2,103 @@
...
+
+### Scenario 1.3b - Access all subpages on the side menu of the Profile Page:
+**Given**: The user is on the profile page.
+
+**When**: The user clicks on each of the following links in the side menu: "My Account", 
+"My Orders", "My Downloadable Products", "My Wish List", "Address Book", "Account Information" "Stored Payment Methods", "My Product Reviews"
+
+**Then**: The user is directed to the corresponding subpage for each link.
+
+### Scenario 1.3c - Change account information:
```
Comment (Marko-Shevchuk) [inline magento/user_stories.md:37]: Add negative scenarios to 1.3c, 1.3d, 1.3e, 1.3h and others that need a negative counterpart

**Review by Marko-Shevchuk — CHANGES_REQUESTED** [review]

**Comment by Stepan0806** [comment]
Added negative scenarios

File: magento/user_stories.md [hunk 4]
```diff
@@ -4,8 +4,120 @@
...
+
+**Description**:
+- Every page on Magento should have a link to the profile page when the user is logged in.
+- The profile page should contain links to several subpages in the account navigation menu: "My Account", 
+"My Orders", "My Downloadable Products", "My Wish List", "Address Book", "Account Information" "Stored Payment Methods", "My Product Reviews".
+- On the "My Account" subpage users can view their contact information and change their password, default billing address, or default shipping address.
+- On the "My Orders" subpage users can view the table of orders they have made with these columns: Order id | Date | Ship To | Order Total | Status | Action
+- On the "My Downloadable Products" subpage users can view a table of digital products they have bought with these columns: Order id | Date | Title | Download | Status | Remaining downloads
+- On the "My Wish List" subpage users can view a list of products they have liked earlier and order them
+- On the "Adress Book" subpage user can add addresses and choose the default billing address and default shipping address.
```
Comment (Marko-Shevchuk) [inline magento/user_stories.md:17]: Address

File: magento/user_stories.md [hunk 5]
```diff
@@ -4,8 +4,120 @@
...
+
+**When**: The user clicks on the "My Account" link, which is accessible from every page.
+
+**Then**: The user is directed to the profile page.
+
+### Scenario 1.3b - Access all subpages on the account navigation menu of the Profile Page:
+**Given**: The user is on the profile page.
+
+**When**: The user clicks on each of the following links in the account navigation menu: "My Account", 
+"My Orders", "My Downloadable Products", "My Wish List", "Address Book", "Account Information" "Stored Payment Methods", "My Product Reviews"
```
Comment (Marko-Shevchuk) [inline magento/user_stories.md:33]: "Account Information" "Stored Payment Methods",

File: magento/user_stories.md [hunk 6]
```diff
@@ -4,8 +4,120 @@
...
+**When**: The user clicks on the "Download" for a downloadable product.
+
+**Given**: The remaining downloads count equal to zero.
+
+**Then**: A message with text "download limit has been exceeded" pops up
+
+### Scenario 1.3k - Invalid input:
+**When**: The user types invalid name, phone number, card number, email or etc. on any field at profile page.
+
+**Then**: Corresponding field is marked with message "invalid input"
```
Comment (Marko-Shevchuk) [inline magento/user_stories.md:116]: Each corresponding field

**Review by Marko-Shevchuk — APPROVED** [review]

**Pull request merged**